src/components/SignUpPage.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { useNavigate, Link } from 'react-router-dom';
+import { signup } from '../redux/user/userAPI';
+import '../styles/SignUpPage.css';
+
+const SignUp = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const [value, setValue] = useState({
+    name: '',
+    email: '',
+    password: '',
+  });
+
+  const onChange = (e) => {
+    setValue({
+      ...value,
+      [e.target.id]: e.target.value,
+    });
+    console.log(value);
+  };
+
+  const createUser = (e) => {
+    e.preventDefault();
+    dispatch(signup(value, navigate));
+    // navigate('/login');
+    // if (dispatch(signup(value))) {
+    //   navigate('/login');
+    // }
```

Comment:
- [ ] Please, kindly remove this old piece of commented code, to improve the cleanliness of the repo 💪